templates/server/src/main/resources/archetype-resources/core/src/main/resources/config/application.properties

```diff
@@ -21,6 +21,7 @@ spring.datasource.url=jdbc:hsqldb:file:./.${rootArtifactId}
 #elseif ($dbType == 'postgresql')
 spring.datasource.password=todo
 spring.datasource.url=jdbc:postgresql://localhost/db
+spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
```

Comment:
Isnt this PR about a bug with MySQL? So there is also an issue with PostgreSQL?
It would be nice to add a comment with a link to an explanation of the workaround.
This would help a lot to understand your change and also for end-users to get the reason of this "non-standard" config.

Reply:
Maybe this one (just my quick guess):
https://hibernate.atlassian.net/browse/HHH-12368